Extensions/Lighting/lightobstacleruntimebehavior.ts

```diff
@@ -53,7 +53,7 @@ namespace gdjs {
     getAllObstaclesAround(
       object: gdjs.RuntimeObject,
       radius: number,
-      result: gdjs.BehaviorRBushAABB<gdjs.LightObstacleRuntimeBehavior>[]
+      result: gdjs.LightObstacleRuntimeBehavior[]
     ) {
```

Comment:
Same as for the other PR: Adding the return type for this function could have helped